src/view/widgetelement.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import Element from './element.js';
+
+export default class WidgetElement extends Element {
+	constructor( name, attributes, children  ) {
+		super( name, attributes, children  );
+
+		// Set content editable attribute to false for all WidgetElements.
+		this.setAttribute( 'contenteditable', false );
+	}
+
+	/**
+	 * Returns `null` because block filler is not needed.
+	 *
+	 * @returns {null}
+	 */
+	getFillerOffset() {
+		return null;
+	}
+}
```

Comment:
I'm thinking how we will add other widget parts, like drag handler. If it will not be a child in the view and it will appear only in the DOM (added in the rendering/DOM conversion), we may not add `contenteditable` attribute here too. On the other hand, I like that this attribute is here because it's more transparent.